### ISSUE TITLE
The "GenerateDepsFile" task failed unexpectedly

### ISSUE THREAD

**Issue by davidfowl** [dotnet/sdk#1300]
```
git clone https://github.com/aspnet/SignalR/
git checkout ba4fe40b516328602b15163454f7ae703dfbcf18
cd src\Microsoft.AspNetCore.Sockets.Client
dotnet restore
dotnet build
```

```
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018: The "GenerateDepsFile" task failed unexpectedly.\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft
.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018: System.ArgumentException: Can not add property System.Net.WebSockets.Client to Newtonsoft.Json.Linq.JObject
. Property with the same name already exists on object.\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNe
tCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JObject.ValidateToken(JToken o, JToken existing)\r [C:\Users\dfowler\Documents\G
itHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JContainer.InsertItem(Int32 index, JToken item, Boolean skipParentCheck)\r [C:\U
sers\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj
]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JObject.InsertItem(Int32 index, JToken item, Boolean skipParentCheck)\r [C:\User
s\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JContainer.AddInternal(Int32 index, Object content, Boolean skipParentCheck)\r [
C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.cs
proj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JContainer.AddInternal(Int32 index, Object content, Boolean skipParentCheck)\r [
C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.cs
proj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JContainer.Add(Object content)\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\
Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JObject..ctor(Object content)\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\M
icrosoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.AddDependencies(JObject libraryObject, I
Enumerable`1 dependencies)\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsof
t.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.WriteTargetLibrary(Library library)\r [C
:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csp
roj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.<WriteTarget>b__7_0(Library library)\r [
C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.cs
proj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at System.Linq.Enumerable.SelectArrayIterator`2.MoveNext()\r [C:\Users\dfowler\Documents\GitHub\SignalR\
src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JContainer.AddInternal(Int32 index, Object content, Boolean skipParentCheck)\r [
C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.cs
proj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JContainer.Add(Object content)\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\
Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Newtonsoft.Json.Linq.JObject..ctor(Object content)\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\M
icrosoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.WriteTarget(IReadOnlyList`1 libraries)\r
 [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.
csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.WriteTargets(DependencyContext context)\
r [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client
.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.Write(DependencyContext context)\r [C:\U
sers\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj
]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Extensions.DependencyModel.DependencyContextWriter.Write(DependencyContext context, Stream
stream)\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Socket
s.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.NET.Build.Tasks.GenerateDepsFile.ExecuteCore()\r [C:\Users\dfowler\Documents\GitHub\SignalR
\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.NET.Build.Tasks.TaskBase.Execute()\r [C:\Users\dfowler\Documents\GitHub\SignalR\src\Microso
ft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()\r [C:\
Users\dfowler\Documents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.cspro
j]
C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(102,5): er
ror MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__25.MoveNext() [C:\Users\dfowler\Docum
ents\GitHub\SignalR\src\Microsoft.AspNetCore.Sockets.Client\Microsoft.AspNetCore.Sockets.Client.csproj]
    0 Warning(s)
    1 Error(s)
```

```
.NET Command Line Tools (2.0.0-preview2-006341)

Product Information:
 Version:            2.0.0-preview2-006341
 Commit SHA-1 hash:  19d305783b

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Users\dfowler\.dotnet\x64\sdk\2.0.0-preview2-006341\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0-preview1-002111-00
  Build    : 1ff021936263d492539399688f46fd3827169983
```

**Comment by livarcocc** [comment]
Is this consistent @davidfowl? It would be great if it were.

**Comment by dsplaisted** [comment]
This looks like the same thing as #1289 which is fixed by #1290

**Comment by davidfowl** [comment]
@livarcocc yes it is.
